modules/vantagecloud-lake/pages/vantagecloud-lake-demos-visual-studio-code.adoc

```diff
@@ -0,0 +1,149 @@
+= Run Teradata Jupyter Notebook Demos for VantageCloud Lake in Visual Studio Code
+:experimental:
+:page-author: Janeth Graziani
+:page-email: Janeth.graziani@teradata.com
+:page-revdate: January 11, 2024
+:description: Learn how to run VantageCloud Lake Demos using Jupyter notebooks in Visual Studio Code. 
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, teradatasqlalchemy, vantagecloud, vantagecloud lake, public internet, visual studio code, IDE, data analytics, data science
+:dir: vantagecloud-lake-demos-visual-studio-code
+
+== Overview
+Visual Studio Code is a popular open-source code editor compatible with Windows, MacOs, and Linux. Developers use this Integrated Development Environment (IDE) for coding, debugging, building, and deploying applications. In this quickstart guide, we launch VantageCloud Lake Jupyter notebook demos within Visual Studio Code. 
+
+image::{dir}/vscode.png[vscode.png,align="center", width=80%]
+
+== Prerequisites
+Before you begin, ensure you have the following prerequisites in place:
+
+* https://www.docker.com/products/docker-desktop/[Docker Desktop] installed
+* https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] installed
+** Required to download git repo from https://github.com/Teradata/lake-demos.git
+* https://code.visualstudio.com/download[Visual Studio Code] installed
+* A Teradata VantageCloud Lake account with organization URL and login details from the Teradata welcome letter
+** Once logged in follow these https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html#_create_an_environment[intructions] to create a VantageCloud Lake Enviorment
+
+== Clone VantageCloud Lake Demo repository 
+Begin by cloning the GitHub repository and navigating to the project directory:
+[source, bash]
+----
+git clone https://github.com/Teradata/lake-demos.git
+cd lake-demos
+----
+== Mount Docker files 
```

Comment:
In this section we start a docker container, not only mounting the files. I suggest the following title for the section:

```suggestion
== Start a Jupyterlab docker container with Teradata Jupyter Exensions
```